backend/scripts/fix_kb_retriever_and_reindex.py

```
@@ -0,0 +1,503 @@
+            trigger_summary=trigger_summary,
+        )
+        log_success(
+            f"Celery task queued: document id={document.id}, task_id={celery_task.id}"
+        )
+        return {
+            "success": True,
+            "celery_task_id": celery_task.id,
+            "index_generation": index_generation,
+            **details,
+        }
+    except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)